src/f32-argmaxpool/f32-argmaxpool-4x-rvv-c1v.c

```diff
@@ -0,0 +1,73 @@
+// Copyright 2024 Imagination Technologies, Inc.
+//
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree.
+
+#include <assert.h>
+#include <riscv_vector.h>
+
+#include "xnnpack/argmaxpool.h"
+
+void xnn_f32_argmaxpool_ukernel_4x__rvv_c1v(
```

Comment:
we use 'u1v' for kernels that use 'm1' meaning unrolled to 1 vector.
Normally we'd make this into a template and generate versions for m1, m2, m4 and maybe m8 to allow some performance testing with different hardware and later put the best of them in the config.
the filename should also be -u1v.c

Reply:
Ok. I will create a multiple LMUL versions and update it in separate PR.

Reply:
done